src/CMakeLists.txt

```diff
@@ -504,6 +504,7 @@ target_link_libraries(mudlet
     ${PUGIXML_LIBRARIES}
     communi
     edbee-lib
+    -lstdc++
```

Comment:
This is a separate PR https://github.com/Mudlet/Mudlet/pull/2833, included here so I can build locally.